src/components/details/Details.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { Link, useParams } from 'react-router-dom';
+import { AiOutlineSetting } from 'react-icons/ai';
+import { TfiArrowCircleRight } from 'react-icons/tfi';
+import { BiLeftArrow } from 'react-icons/bi';
+import './details.css';
+import { baseURL } from '../../helpers/api';
+
+function Details() {
```

Comment:
- [ ] On the "details" page for each car, this back button is not working for me:

![image](https://user-images.githubusercontent.com/57421823/223581137-1e6d8ab5-e72e-4bcb-9859-e99c30e6bac7.png)

Kindly make it so it navigates to the main page.   That way, your UX and user experience will be vastly improved. 👍